docs/05/d.md

```diff
@@ -34,7 +34,7 @@ Here's [the typeclass contract for `MonadFilter`][MonadFilterSource]:
 }
 ```
 
-This enables a derivative function/operator `filter` and `filterM`:
+This enables the derivative functions/operators `filter` and `filterM`:
```

Comment:
So you want to retain both. Got it. But how do you want to go with the number agreement issue?